integration/src/integration/java/pl/allegro/tech/hermes/integration/env/SpringlessServiceIteratorProvider.java

```diff
@@ -0,0 +1,38 @@
+package pl.allegro.tech.hermes.integration.env;
+
+import org.glassfish.jersey.internal.ServiceFinder;
+import org.glassfish.jersey.server.spring.SpringComponentProvider;
+
+import java.util.Iterator;
+import java.util.Spliterator;
+import java.util.Spliterators;
+import java.util.stream.StreamSupport;
+
+class SpringlessServiceIteratorProvider extends ServiceFinder.ServiceIteratorProvider {
```

Comment:
\[Checkstyle\] ERROR: Class SpringlessServiceIteratorProvider should be declared as final\.